scripts/bosh-cli.sh

```diff
@@ -26,4 +26,4 @@ docker run \
     --env "BOSH_ENVIRONMENT=bosh.${SYSTEM_DNS_ZONE_NAME}" \
     --env "BOSH_CA_CERT" \
     --env "BOSH_DEPLOYMENT=${DEPLOY_ENV}" \
-    governmentpaas/bosh-shell:0eff5b6a9c092f865a2b19cc4e75a3b539b82fa2
+    governmentpaas/bosh-shell:4012d25ceb903b46908a830b8e05773ced1c8f86
```

Comment:
Ditto here, this is still using an old version (2.0.48)...

Reply:
See: https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/135